app/controllers/provider/admin/registry/policies_controller.rb

```diff
@@ -68,7 +68,7 @@ def policy_params
   end
 
   def authorize_policies
-    authorize! :manage, :policy_registry
+    authorize! :manage, :policy_registry_ui
```

Comment:
I don't know if it should have both or only this one for the real usage later on, but right now it is going to be false anyway so for now it is ok 👍

Reply:
See the rolling update, it is inheriting from `:policy_registry`

Reply:
ah right sorry, i had not noticed. Good one 👍